timesketch/lib/analyzers/sequence_sessionizer.py

```diff
@@ -0,0 +1,152 @@
+"""Sequenced activity sessionizing sketch analyzer plugin."""
+
+from __future__ import unicode_literals
+
+#from timesketch.lib.analyzers import manager
+from timesketch.lib.analyzers import sessionizer
+
+
+class SequenceSessionizerSketchPlugin(
+        sessionizer.SessionizerSketchPlugin):
+    """Base class for specialized sequenced activity sessionizings
+    sketch plugins.
+
+    Attributes:
+        event_seq_name: The name of the event sequence.
+        event_seq: List of dictionaries of attributes describing the events
+        event_storage: List of events.
+        num_event_to_find: Number that shows which event from the sequence
+        should be matched.
```

Comment:
Please indent with 4 spaces when doing a newline in the `Args` section
```suggestion
            should be matched.
```